conans/client/cache/cache.py

```diff
@@ -265,25 +263,13 @@ def generators(self):
         return generators
 
     def initialize_config(self):
+        # TODO: This is called by ConfigAPI.init(), maybe move everything there?
         if not os.path.exists(self.conan_conf_path):
-            save(self.conan_conf_path, normalize(get_default_client_conf()))
-
-    def reset_config(self):
-        if os.path.exists(self.conan_conf_path):
```

Comment:
This belongs to ``ConfigAPI``, not to ``Cache``.

I have left only the ``initialize_config()``, as too many tests and other behavior relies on it, it would be a big change (we can do it later if we think so)